ajax/libs/moment-recur/1.0.5/moment-recur.js

```diff
@@ -0,0 +1,744 @@
+(function (root, factory) {
+    if (typeof exports === 'object') {
+        module.exports = factory(require('moment'));
+    } else if (typeof define === 'function' && define.amd) {
+        define('moment-recur', ['moment'], factory);
+    } else {
+        root.moment = factory(root.moment);
+    }
+}(this, function (moment) {
+    var hasModule;
+
+    hasModule = (typeof module !== "undefined" && module !== null) && (module.exports != null);
+
+    if (typeof moment === 'undefined') {
+      throw Error("Can't find moment");
+    }
+
+    // Interval object for creating and matching interval-based rules
+    var Interval = (function() {
+        function createInterval(units, measure) {
+            // Make sure all of the units are integers greater than 0.
+            for (var unit in units) {
+                if (units.hasOwnProperty(unit)) {
+                    if (parseInt(unit, 10) <= 0) {
```

Comment:
https://github.com/c-trimm/moment-recur/blob/v1.0.5/moment-recur.js#L24
```js
                     if ( parseInt(unit, 10) <= 0 ) {
```


Obviously different indent here ... @cdnjs/intern3 you should use md5 and sha to make sure the main files are 100% the same as upstream.